lib/core-integration/src/Test/Integration/Scenario/CLI/Shelley/Transactions.hs

```diff
@@ -1064,13 +1064,15 @@ spec = do
           , (Just validTime1, Just validTime2)
           ]
 
-      longAddr = replicate 10000 '1'
       encodeErr = "Unable to decode Address:"
       parseErr = "Parse error. Expecting format \"<amount>@<address>\""
       matrixInvalidAddrs =
-          [ ( "long hex", longAddr, encodeErr )
-          , ( "short hex", "1", encodeErr )
-          , ( "-1000", "-1000", encodeErr )
+-- TODO: For the haskell node, hex is valid. For jormungandr it is not.
+--  longAddr = replicate 10000 '1'
+--  We should optimally find a way to test this.
+--          [ ( "long hex", longAddr, encodeErr )
+--          , ( "short hex", "1", encodeErr )
+          [ ( "-1000", "-1000", encodeErr )
```

Comment:
What is this matrix doing here anyway :thinking: ? We now have `Cardano.Wallet.Api.Malformed` to exercise parsing errors on the API data-types.

Reply:
Probably because it is CLI, so to verify that the error messages from the API are displayed correctly there, and also this error:
```
parseErr = "Parse error. Expecting format \"<amount>@<address>\""
```
does not fall into that category.